src/main/java/http/cookie/CookieParser.java

```diff
@@ -0,0 +1,27 @@
+package http.cookie;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CookieParser {
+
+    private CookieParser() {
+    }
+
+    public static final int KEY_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
```

Comment:
```suggestion
    public static final int KEY_INDEX = 0;
    public static final int VALUE_INDEX = 1;
    
    private CookieParser() {
    }
```
아래의 링크를 보면 초기화 순서 가이드를 확인할 수 있습니다~ 

>https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace